### PR TITLE
Use a different attribute, and various minor upgrades

### PR DESCRIPTION
**Pull request by domenic**
I've tested this and it seems to work. Once it's merged, I can do a v1.0.0 release.

@tobie, could you give us npm publish access as well?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,20 +3,12 @@
   "version": "0.4.2",
```
Comment (annevk) [inline package.json:3]: Is it okay to leave this like this? It's a nice version number, but we are making a breaking change.

**Review by annevk — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (domenic) [reply package.json:3]: We should update it in a followup commit, but it's slightly easier to do that separately for npm-related reasons. (At least when you have push-to-main privileges.)

**Comment by tobie** [comment]
What npm login names should I give access to?

**Comment by domenic** [comment]
I see you found my npm name; thank you!

**Pull request merged**